includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -117,6 +134,8 @@ function ( WC_Payment_Gateway_WCPay $gateway ) {
 	public function update_settings( WP_REST_Request $request ) {
 		$this->update_is_wcpay_enabled( $request );
 		$this->update_enabled_payment_methods( $request );
+		$this->update_is_digital_wallets_enabled( $request );
+		$this->update_digital_wallets_enabled_sections( $request );
```

Comment:
Same comment as above re "sections" vs "locations" (I'm not going to repeat it for every repetition, tho :D )